Formula/pulumi.rb

```diff
@@ -0,0 +1,30 @@
+class Pulumi < Formula
+  desc "Define cloud applications and infrastructure in your favorite language"
+  homepage "https://pulumi.io/"
+  url "https://github.com/pulumi/pulumi.git",
+      :tag => "v0.15.0",
+      :revision => "23cbfa503d856f57a3e00599d53c04fe69c93cf0"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/pulumi/pulumi"
+    dir.install buildpath.children
+    (buildpath/"bin").mkpath
```

Comment:
What's happening here to necessitate this?

Reply:
It's a pretty standard set of operations for a `go`-using formula ? I don't really get what is bothering you sorry 😞